### ISSUE TITLE
Phonotactic probability question from a user

### ISSUE THREAD

**Issue by kchall** [PhonologicalCorpusTools/CorpusTools#392]
e-mail from a user:

Hello, I'm writing to inform you of a small documentation problem with the PCT in the phonotactic probability section as regards the .corpus format. All of the examples of transcriptions that you provide are of the format "letter.letter" (e.g., "h.e.l.l.o"). Following that example a single letter would be without a '.' (e.g., "I", instead of "I."). When calculating bigram probability his leads the GUI to an error: "division by zero", which I assume is to mean that it does not recognize a letter unless it is followed by the '.'

In testing this I found that the '.' does indeed need to follow every letter in order for the program to calculate the phonotactic probability of those bigrams, meaning that "h.e.l.l.o" would not calculate the final "o", but "h.e.l.l.o." would.

If I am somehow mistaken, please do let me know. Otherwise, I hope I have helped out a little bit. It did take me some time to figure out the problem, which could easily be fixed with a short paragraph that explained the .corpus format.

**Comment by mmcauliffe** [comment]
It seems, if I'm parsing this right, as though they imported '.' delimited transcriptions without a delimiter specified, so maybe the segment '.' is being treated as a segment?  Do you think you could email them back and see if they'd be willing to share the corpus file they're using for us to debug it, as well as the actual output results that are anomalous?

On a slightly related note, there's a slight bug in phonotactic probability (more a previously untested edge case) where when biphone probability is calculated for a transcription with a single phone, it raises a ZeroDivisionError.  The way Iphod treats it (and I've fixed develop to do so as well) is that these have 0 for biphone probability.  So that might be worth a bug fix, and it might be related to this user's issue, but it might also not be.

**Comment by kchall** [comment]
Follow-up: "The problem I was experiencing was as you said, encountering an error message when trying to calculate biphone probability with a dataset that has individual phones. The tests that I made I've attached as a text file. Their purpose was to see whether adding a '.' to an individual phone (and thus at the end of each string) would negatively effect the calculation. The second attached txt file is my current dataset. You will find that I calculated biphone probability with a final '.' for each string." 

These test files are in the test files Dropbox folder.
